iOS-sample-app/Showcases/Transactions/TransactionsViewController.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import ContentSquare
+
+struct MyItem
+{
+    private(set) var myTitle: String
+    private(set) var myPrice: Float
+    private(set) var myCurrency: Currency
+
+    init(title: String, price: Float, currency: Currency)
+    {
+        myTitle = title
+        myPrice = price
+        myCurrency = currency
+    }
+}
+
+// This class showcases how to track purchases made by an user with CustomerTransactions
+class TransactionsViewController: UIViewController {
+
+    private var itemsInCart = [MyItem]()
+    private var selectedCurrency : Currency = .eur
+    private var itemCount = 0
+
+    override func loadView()
+    {
+        super.loadView()
+
+        let validateButton = UIButton(type: .custom)
+        validateButton.translatesAutoresizingMaskIntoConstraints = false
+        validateButton.backgroundColor = .cyan
+        self.view.addSubview(validateButton)
+        validateButton.setTitle("Validate Cart", for: .normal)
+        validateButton.setTitleColor(.black, for: .normal)
+        validateButton.bottomAnchor.constraint(equalTo: self.view.bottomAnchor, constant: -50).isActive = true
+        validateButton.centerXAnchor.constraint(equalTo: self.view.centerXAnchor).isActive = true
+        validateButton.addTarget(self, action: #selector(validateCart), for: .touchUpInside)
+
+        let addItemButton = UIButton(type: .custom)
+        addItemButton.translatesAutoresizingMaskIntoConstraints = false
+        addItemButton.backgroundColor = .lightGray
+        self.view.addSubview(addItemButton)
+        addItemButton.setTitle("Add an item to Cart", for: .normal)
+        addItemButton.setTitleColor(.black, for: .normal)
+        addItemButton.bottomAnchor.constraint(equalTo: validateButton.topAnchor, constant: -50).isActive = true
+        addItemButton.centerXAnchor.constraint(equalTo: validateButton.centerXAnchor).isActive = true
+        addItemButton.addTarget(self, action: #selector(addItemToCart), for: .touchUpInside)
```

Comment:
Do you have some reason to prefer coding the screen instead of using a storyboard? I understand there are reasons, especially in large projects. But here I'd like to avoid having a lot of screen creation in code. It distracts the reader, who's here to learn how to make calls to the SDK, from SDK related code.

Reply:
It's just a personal preference, but it's true that the UI code has no interest for developers discovering our SDK